src/template/script/generate

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+APP=$1
+STACK=$2
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ROOT_DIR="${DIR}/.."
+OUTPUT_DIR="${ROOT_DIR}/cdk.out"
+
+
+if [[ -z $APP ]]; then
+  echo -e "\033[31mPass the app you'd like to generate Cloudformation for to this script"
+  exit 1
+fi
+
+pushd "${ROOT_DIR}" > /dev/null || exit
+yarn --silent cdk synth --app="ts-node ${ROOT_DIR}/bin/${APP}.ts"  -o "${OUTPUT_DIR}" --path-metadata false --version-reporting false ${STACK}
```

Comment:
minor: IIUC when a CDK project is generating a single stack, we can call `cdk synth` with no arguments as it defaults to the values from the `cdk.json` file. That is, we could simplify this file.

Reply:
Ah, interesting. One of the things I'd like to do next is more specific setup for either single or multiple app projects so maybe that's something I can look at then. Having said that, I've been thinking that maybe we could keep the structure more consistent so maybe one to discuss.